docker/Dockerfile.core

```diff
@@ -22,10 +22,15 @@ RUN apt-get -y update > /dev/null && \
 
 COPY scripts/fix-vulnerabilities.sh /tmp
 
-RUN apt update -y
-RUN apt-get -y --no-install-recommends install openjdk-17-jdk
+RUN apt-get -y update > /dev/null && \
+    apt-get -y --no-install-recommends install \
+    openjdk-17-jre-headless \
```

Comment:
Does this need to be seperated from the install command above? Is openjdk-17-jre-headless updated so much more often?